### PR TITLE
Fix assembly error of image URL

### PR DESCRIPTION
**Pull request by hythof**
### Purpose/goal of Pull Request (Issue #)
Fix wrong assemble URL when pathname parameter is empty.

### Comments
x http://hostname:8888image.png
o http://hostname:8888/image.png

### CODE REVIEW

File: lib/wovnrb/html_replacers/image_replacer.rb [hunk 1]
```diff
@@ -20,9 +20,9 @@ def replace(dom, lang)
           if src !~ /:\/\//
             # if this is a path with a leading slash
             if src =~ /^\//
-              src = "#{@url[:protocol]}://#{@url[:host]}#{src}"
+              src = File.join("#{@url[:protocol]}://#{@url[:host]}", src)
```
Comment (a-bougouin) [inline lib/wovnrb/html_replacers/image_replacer.rb]: It is a bit weird to use `File.join` on url

File: test/lib/html_replacers/image_replacer_test.rb [hunk 2]
```diff
@@ -69,10 +69,19 @@ def test_replace_root_path
     end
 
     def test_replace_absolute_path
+        img_test_helper('/hello/')
+    end
+
+    def test_replace_empty_path
+        img_test_helper('')
```
Comment (mrasu) [inline test/lib/html_replacers/image_replacer_test.rb]: please write assertion in test's method to easy to understand what is expected

File: lib/wovnrb/html_replacers/image_replacer.rb (hunk 1 above)
Comment (hythof) [inline lib/wovnrb/html_replacers/image_replacer.rb]: I think so too.  Fix it.

File: test/lib/html_replacers/image_replacer_test.rb (hunk 2 above)
Comment (hythof) [inline test/lib/html_replacers/image_replacer_test.rb]: That way is easier to understand. Fix it.

**Pull request merged**